src/CEOFESABundle/Controller/SessionController.php

```diff
@@ -739,17 +740,29 @@ public function editPresenceAjaxAction(Request $request){
 
         $form = $this->createForm(new PresenceType(), $presence);
 
-        $form->handleRequest($request);
-        if ($form->isValid()) {
+        if ($form->handleRequest($request)->isValid()) {
+            if($presence->getPscSession()->getSesDuree()<$form->get('pscDuree')->getData()) {
+                $form->get('pscDuree')
+                     ->addError(new FormError("La durée doit être inferieur ou égale à celle de la session"));
+
+                return array(
+                    'presence'  => $presence,
+                    'form'      => $form->createView(),
+                );
+            }
+
             $em->persist($presence);
             $em->flush();
 
-            return $this->render("::Session\presence.html.twig", array('presence' => $presence));
+            return $this->render(
+                '::Session\presence.html.twig',
+                array('presence' => CeoHelper::DurationFloatToArray($presence->getPscDuree()))
+            );
```

Comment:
return $this->render('::Session\presence.html.twig', array(
                'presence' => CeoHelper::DurationFloatToArray($presence->getPscDuree()),
            ));